charts/llm-inference-app/templates/externalService.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.autoscaling.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: keda-add-ons-http-interceptor-proxy-app
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: ExternalName
+  externalName: keda-add-ons-http-interceptor-proxy.platform.svc.cluster.local
```

Comment:
and this one, since app-types will give this name

Reply:
I actually think this FQDN will be the same across all of the clusters, we always deploy to the platform namespace, don't we?
Why do you think it should be parametrized?